includes/admin/pages.php

```diff
@@ -272,7 +272,10 @@ public function html()
     		}
     	}
     	// It will display when the Pro is activated.
-    	if (!is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php')) { ?>
+    	if (
+    		!is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php') &&
```

Comment:
We don't need this condition. Only the following state should be enough. Does this mean everybody who got the release earlier was not getting the correct message?

Reply:
Can we check for class name instead of folder?

Reply:
I see folder as `simple-calendar-google-calendar-pro` in our demo.

Reply:
I have tried with class name but it is not working may be due to SC is loaded before the pro addon. And this is actually a right way to check plugins exist in WordPress. 
I am not sure why he use the "main" in plugin folder maybe he got the folder name like this when he clone from git so I have update the condition.

Reply:
on check on demo staging site and it is working as expected.
https://www.screenpresso.com/=o7v6e